core/templates/domain/skill/skill-backend-api.service.spec.ts

```diff
@@ -16,61 +16,54 @@
  * @fileoverview Unit tests for SkillBackendApiService.
  */
 
-require('domain/editor/undo_redo/undo-redo.service.ts');
-require('domain/skill/skill-backend-api.service.ts');
-require('services/csrf-token.service.ts');
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// the code corresponding to the spec is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
-
-
-describe('Skill backend API service', function() {
-  var SkillBackendApiService = null;
-  var UndoRedoService = null;
-  var $httpBackend = null;
-  var sampleResponse = null;
-  var CsrfService = null;
-  var sampleResponse2 = null;
-
-  beforeEach(angular.mock.module('oppia'));
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    var ugs = new UpgradedServices();
-    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
-      $provide.value(key, value);
-    }
-  }));
+import { HttpClientTestingModule, HttpTestingController } from
+  '@angular/common/http/testing';
+import { TestBed, fakeAsync, flushMicrotasks } from '@angular/core/testing';
+
+
+import { SkillBackendApiService } from
+  'domain/skill/skill-backend-api.service.ts';
+import { CsrfTokenService } from 'services/csrf-token.service.ts';
```

Comment:
Add these in imports in sorted order. (Also, you don't need to end them with `.ts`)

Reply:
Done!